lib/merch_calendar/stitch_fix_fiscal_year_calendar.rb

```diff
@@ -40,8 +50,29 @@ def start_of_quarter(year, quarter)
         start_of_month(year, 10)
       end
     end
+    
+    # Returns the quarter that the merch month falls in
+    #
+    # @param [Fixnum] merch month
+    # @return [Date] the quarter that the merch_month falls in
+    def quarter(merch_month)
+      case merch_month
+      when 1,2,3
+        return QUARTER_1
+      when 4,5,6
+        return QUARTER_2
+      when 7,8,9
+        return QUARTER_3
+      when 10,11,12
+        return QUARTER_4
+      end
```

Comment:
It's a good practice to always have an `else` for your case statements.  Otherwise, the behavior is ambiguous. 

I generally raise an error if I trigger the `else` condition.

Reply:
this would be consistent with the other raise logic too.

Reply:
Should I type:
```ruby
    def quarter(merch_month)
      case merch_month
      when 1,2,3
        return QUARTER_1
      when 4,5,6
        return QUARTER_2
      when 7,8,9
        return QUARTER_3
      when 10,11,12
        return QUARTER_4
     else
        nil
      end
    end
    
```
OR
```ruby
    def quarter(merch_month)
      case merch_month
      when 1,2,3
        return QUARTER_1
      when 4,5,6
        return QUARTER_2
      when 7,8,9
        return QUARTER_3
     else
        return QUARTER_4
      end
    end
    
```

Reply:
neither :)

```
else
  raise "invalid merch month"
end
```

Reply:
got it!